pkg/interface/src/views/apps/publish/components/NoteNavigation.tsx

```diff
@@ -30,45 +31,60 @@ function NavigationItem(props: {
   );
 }
 
+function getAdjacentId(graph: Graph, child: number, backwards = false): number | null {
```

Comment:
I wonder if we should make a scry to receive a list of all the IDs in the graph. Would make this easier to paginate.

Reply:
Good idea